src/pubsub.c

```diff
@@ -101,12 +104,12 @@ pubsubtype pubSubShardType = {
  * message. However if the caller sets 'msg' as NULL, it will be able
  * to send a special message (for instance an Array type) by using the
  * addReply*() API family. */
-void addReplyPubsubMessage(client *c, robj *channel, robj *msg) {
+void addReplyPubsubMessage(client *c, robj *channel, robj *msg, robj *message_bulk) {
```

Comment:
don't we also need to patch `addReplyPubsubPatMessage` and reply with `spmessage`?

Reply:
No, sharded pubsub doesn't support pattern matching.

Reply:
We decided that sharded pubsub wouldn't include patterns since we don't have a canonical way to route patterns in cluster mode. We can implement in the future as a non-breaking change though.